### DISCUSSION TITLE
Why did the CSB url's change from id.csb.app to id.sse.codesandbox.io?

### DISCUSSION THREAD

**Discussion by Explosion-Scratch** [codesandbox/codesandbox-client#5274] [Q&A]
Title. .csb.app was shorter.

**Comment by garethx** [comment]
Nothing changed recently, so perhaps you've just started using container sandboxes and noticed the difference from client ones. There's no great reason for the difference, mostly just that the container sandboxes are served from different servers than the client ones, but note that the csb.app URLs work for all sandboxes. So you can also use id.csb.app for id.sse.codesandbox.io sandboxes too.
